### ISSUE TITLE
RangeError: Invalid string length

### ISSUE THREAD

**Issue by Mave95** [janoside/btc-rpc-explorer#486]
The following tx is causing trouble in a couple of bitcoin software projects
[73be398c4bdc43709db7398106609eea2a7841aaf3a4fa2000dc18184faa2a7e](https://mempool.space/tx/73be398c4bdc43709db7398106609eea2a7841aaf3a4fa2000dc18184faa2a7e)

**Environment (please complete the following information):**

 - Bitcoin Core: v23
 - NodeJS Version: 18
 - Browser: Chrome
 - Code Version / Commit: [863f7c4129](https://github.com/janoside/btc-rpc-explorer/commit/863f7c4129)
 - Installation Method: source code
 
**To Reproduce**

https://bitcoinexplorer.org/tx/73be398c4bdc43709db7398106609eea2a7841aaf3a4fa2000dc18184faa2a7e

**Screenshots or Log Output**

```
2022-11-01T11:56:50.534Z btcexp:error Error ExpressUncaughtError: RangeError: Invalid string length, json: {}, userData: [object Object] (json: {})
2022-11-01T11:56:50.536Z btcexp:errorVerbose Stack: RangeError: Invalid string length
    at Object.pug_interp [as cardHighlightBadge] (eval at wrap (/workspace/node_modules/pug-runtime/wrap.js:6:10), <anonymous>:1678:21)
    at Object.eval (eval at wrap (/workspace/node_modules/pug-runtime/wrap.js:6:10), <anonymous>:1707:33)
    at Object.pug_interp [as hexDataDisplay] (eval at wrap (/workspace/node_moduntime/wrap.js:6:10), <anonymous>:1735:4)
    at eval (eval at wrap (/workspace/node_modules/pug-runtime/wrap.js:6:10), <anonymous>:3485:29)
    at eval (eval at wrap (/workspace/node_modules/pug-runtime/wrap.js:6:10), <anonymous>:3498:4)
    at block (eval at wrap (/workspace/node_modules/pug-runtime/wrap.js:6:10), <anonymous>:3551:4)
    at Object.pug_interp (eval at wrap les/pug-runtime/wrap.js:6:10), <anonymous>:1735:4)
    at eval (eval at wrap (/workspace/node_modules/pug-runtime/wrap.js:6:10), <anonymous>:3485:29)
    at eval (eval at wrap (/workspace/node_modules/pug-runtime/wrap.js:6:10), <anonymous>:3498:4)
    at block (eval at wrap (/workspace/node_modules/pug-runtime/wrap.js:6:10), <anonymous>:3551:4)
    at Object.pug_interp (eval at wrap (/workspace/node_modules/pug-runtime/wrap.js:6:10), <anonymous>:66:10)
    at block (eval at wrap (/workspace/node_modules/pug-runtime/wrap.js:6:10), <anonymous>:3452:30)
    at Object.pug_interp (eval at wrap (/workspace/node_modules/pug-runtime/wrap.js:6:10), <anonymous>:1540:10)
    at eval (eval at wrap (/workspace/node_modules/pug-runtime/wrap.js:6:10), <anonymous>:3450:23)
    at template (eval at wrap (/workspace/node_modules/pug-runtime/wrap.js:6:10), <anonymous>:3793:7)
    at Object.exports.renderFile (/workspace/node_modules/pug/lib/index.js:454:38)
    at Object.exports.renderFile (/workspace/node_modules/pug/lib/index.js:444:21)
    at exports.__express (/workspace/node_modules/pug/lib/index.js:493:11)
    at View.engine (/workspace/app.js:175:23)
    at View.render (/workspace/node_modules/express/lib/view.js:135:8)
```

**Comment by jsarenik** [comment]
On be.anyone.eu.org I did a [hotfix](https://be.anyone.eu.org/tx/73be398c4bdc43709db7398106609eea2a7841aaf3a4fa2000dc18184faa2a7e) to redirect this tx to mempool.space :-}

**Comment by janoside** [comment]
Put some fixes in aefce7b3d4ecd2
